src/PortingAssistant.Client.Client/PortingAssistantClient.cs

```diff
@@ -98,7 +98,7 @@ public List<PortingResult> ApplyPortingChanges(PortingRequest request)
                     .Select(recommendation =>
                     {
                         var packageRecommendation = (PackageRecommendation)recommendation;
-                        return new Tuple<string, string>(packageRecommendation.PackageId, packageRecommendation.TargetVersions.First());
+                        return new Tuple<string, Tuple<string, string>>(packageRecommendation.PackageId, new Tuple<string, string>("1.0.0", packageRecommendation.TargetVersions.First()));
```

Comment:
Please check this.. it has 1.0.0 as an original version?

Reply:
Fix this issue